knowledge-base/ShiftLeftSecurity/scan-action/action-security.yml

```diff
@@ -0,0 +1,15 @@
+name: "Security and Licence Scan"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  is-default: false
+  permissions:
+    pull-requests: write
+    pull-requests-reason: to add the scan summary as a comment in pull request # NOTE: https://slscan.io/en/latest/integrations/github-actions/#automatic-pull-request-comments
```

Comment:
@h0x0er the reason goes into the workflow file, so it should not be too long. In this case, can you please make it `to add comments in the pull request`?

@Devils-Knight please add a test for length of the reason - only for permission reasons, not outbound-endpoint reasons. If the length of the reason is more than 50, the test should fail.

Reply:
@varunsh-coder thanks for pointing the `length issue` I had fixed it.
And now the 2nd thing; as per [here](https://slscan.io/en/latest/integrations/github-actions/#automatic-pull-request-comments); it can add a comment in PR to show the scan results